### ISSUE TITLE
Training datasets

### ISSUE THREAD

**Issue by zyy0822** [navervision/lincir#9]
In the paper, CC3M and 2.47M StableDiffusion prompts are employed for training. However, in the released code, three datasets are adopted, so i want to know if the ``'dataset3': 'Geonmo/midjourney-prompts-onlyonly'`` is used for training, i.e.,https://github.com/navervision/lincir/blob/28943db28b4f65d41dc2724b6e79596b0b8cc82d/loader.py#L219C19-L219C21

**Comment by geonm** [comment]
As described in our paper, we trained our models using only Dataset1 (GCC3M captions) and Dataset2 (SDP).

However, upon further evaluation, we found that incorporating Dataset3 resulted in a slight improvement in performance.

So, we included Dataset3 in the version of the code we released.

Using LinCIR, adding extra text datasets to train a model for CIR becomes remarkably straightforward. Therefore, if you aim to enhance the model further, we suggest browsing HuggingFace to discover suitable text datasets. Text datasets, being significantly easier to gather for machine learning than datasets from other modalities, are available in abundance.

As outlined in our paper, we strongly recommend seeking out text datasets rich in keywords, such as nouns. For instance, the [SAM-LLaVA-Captions](https://huggingface.co/datasets/PixArt-alpha/SAM-LLaVA-Captions10M) appears to be a good choice.

**Comment by Pefect96** [comment]
Thank you for your reply! I tried to reproduce the whole training, but found that the performance did not reach the level shown in the paper, and it would not increase after about 5000 steps, and it might decline in the following iteration, and the performance improvement was not obvious compared with the beginning, may I ask if the learning rate has been maintained at 1e-4? Whether adjusting the learning rate can further improve performance.
